Week1/mardown.md

```diff
@@ -0,0 +1,20 @@
+
+#Describe shared testing activities for web and mobile?
+    -They both share tools which are commonly used like:
+     -Usability testing and Performance testing
+    -internet Connectivity testing
+
+# Describe unique testing 
+activities per platform (web and mobile)?
+    
+    -Mobile application need mobile devices for testing , where as , we can test web application on laptops and browsers.
+    -Mobile application has to be installed on mobile devices for that you need storage in devices whereas, we don't need much space to launch web application.
+    -Mobile application are built for wide range of user and so needs to be tested on different screen size, where as, web application can be tested on desktop.
```

Comment:
Important to note that Web apps might be accessed on diffrent browsers and diffrent screen sizes though, there's quite a lot of odd stuff that Safari does for instance (the black sheep of the browser family now Internet Explorer is dead) and that might include testing on mobile browsers. Also emulators and paid for services are available for mobile app testing without a phsyical device